packages/react-native-client/src/hooks/useCamera.ts

```diff
@@ -255,6 +217,36 @@ export function useCamera() {
     [],
   );
 
+  /**
+   * @deprecated
+   * toggles encoding of a video track on/off
+   * @param encoding encoding to toggle
+   */
+  const toggleVideoTrackEncoding = useCallback(
+    async (encoding: TrackEncoding) => {
+      const videoSimulcastConfig =
+        await RNFishjamClientModule.toggleVideoTrackEncoding(encoding);
+      setSimulcastConfig(videoSimulcastConfig);
+    },
+    [],
+  );
+
+  /**
+   * @deprecated
```

Comment:
**question:** If those are deprecated? Can we remove them? Or do we need it for some reason?

Reply:
I'm keeping them for now, as I still need to work out how we will configure stream quality. Once this will be done, we can update/remove this method according to our needs